docs/ios/troubleshooting/mtouch-errors.md

```diff
@@ -1885,11 +1885,11 @@ This is always a bug in Xamarin.iOS; please file a new issue on [github](https:/
 
 <a name="MT4116" />
 
-### MT4116: Could not register the assembly '*': *
+### MT4116: Could not register the assembly '\*': \*
 
 <a name="MT4117" />
 
-### MT4117: The registrar found a signature mismatch in the method '*.*' - the selector indicates the method takes * parameters, while the managed method has * parameters.
+### MT4117: The registrar found a signature mismatch in the method '\*.\*' - the selector indicates the method takes \* parameters, while the managed method has \* parameters.
```

Comment:
Not sure if this was rendering incorrectly or was supposed to be a bolded period

Reply:
I think your fix is correct - it's intending to indicate a wildcard namespace.class